src/user_input.rb

```diff
@@ -0,0 +1,37 @@
+require 'fileutils'
+require 'json'
+require './Items/game'
+require './author'
+
+class UserInput
+  SAVE_DATA = './Json/'.freeze
+  def self.write_game(games)
+    return if games.empty?
+
+    path_file = "#{SAVE_DATA}game.json"
+    data_games = games.map do |game|
+      {
+        id: game.id,
+        title: game.title,
+        multiplayer: game.multiplayer,
+        last_played: game.last_played,
```

Comment:
Please add the author of the game as this is a crucial part of implementing the associations between classes :+1: